.travis.yml

```diff
@@ -13,6 +14,7 @@ cache:
 before_install:
     - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.22.5
     - export PATH=$HOME/.yarn/bin:$PATH
+    - export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
Can we drop this option from the scripts in package.json after this change? Or do the environment options get blown away during build execution?

Reply:
Docs say enviornment variables can be accessed at any stage in the build process. But we still need the option there in the package.json for self hosting right?

Reply:
Great point!